### DISCUSSION TITLE
grid numbering convention

### DISCUSSION THREAD

**Discussion by jbensabat** [NCAR/wrf_hydro_nwm_public#708] [General]
Hello
I am constructing a WRFHYDRO model and I popped into an issue that I would like to clarify.
how is the numbering convention of the grids ? in order words does I=1, J=1 correspond to the South - West corner or to the North-West corner ? in geogrid data are according to I=1, J=1 corresponding to the SW corner. However, in some of the tests shipped with the code (such as the crotton test case, I=1, J=1 corresponds to the North-West corner.
please your help
best
jac

**Comment by rcabell** [comment]
Hi @jbensabat -- this is a Discussion question, not an Issue, so I've moved it to the GitHub Discussions section of the WRF-Hydro repository. Please use this space for future questions, or send them via email to wrfhydro@ucar.edu. 

Please only open Issues for bugs (or feature requests) in the code, and use the template provided when doing so. Thanks!
